laterpay/asset_sources/js/laterpay-backend-contributions.js

```diff
@@ -58,6 +63,13 @@
                         }, 1500)
                     );
 
+                    var onchangeCallback = function () {
+                        resetGenerateButton();
+                        updateLivePreview( undefined, undefined, 'multiple' );
+                    };
+
+                    $o.dialogHeader.on( 'change', onchangeCallback );
```

Comment:
```suggestion
                    $o.dialogHeader.on( 'keyup', onchangeCallback );
```
Could we update it to `keyup` instead, merchant could see the update on preview as they type.

Reply:
Addressed.